app/src/main/java/com/taylorgirard/comicconvo/activities/IndividualMessageActivity.java

```diff
@@ -145,6 +149,57 @@ public void done(Boolean object, ParseException e) {
                         }
                     }
                 });
+
+                ParseQuery<ParseUser> query = new ParseQuery<ParseUser>(ParseUser.class);
+                ParseUser checkMatch = null;
+                try {
+                    checkMatch = query.get(match.getObjectId());
+                } catch (ParseException e) {
+                    e.printStackTrace();
+                }
```

Comment:
This should theoretically never throw an exception because the entire message page would not be loaded if the matched user does not exist. The app does not include functionality for deleting a profile, so the matched user should always exist in practice